app/gateways/http/transfer/get.go

```diff
@@ -15,14 +18,23 @@ type GetTransferResponse struct {
 }
 
 func (h Handler) Get(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
 	accountID := r.Context().Value(http_server.AccountID).(string)
 
-	transferList, err := h.useCase.GetTransfersByAccountID(accountID)
+	transferList, err := h.useCase.GetTransfersByAccountID(ctx, accountID)
 	if err != nil {
-		h.logger.Err(err).
-			Int("status_code", http.StatusBadRequest).
-			Msg("error occurred when try get transfers")
-		http_server.ResponseError(w, http.StatusBadRequest, ErrUnexpected)
+		switch err {
+		case usecase.ErrInvalidId:
+			h.logger.Err(err).
+				Int("status_code", http.StatusBadRequest).
+				Msg("error occurred when try to get transfers")
+			http_server.ResponseError(w, http.StatusBadRequest, ErrInvalidId)
+		default:
+			h.logger.Err(err).
+				Int("status_code", http.StatusBadRequest).
+				Msg("error occurred when try to get transfers")
+			http_server.ResponseError(w, http.StatusBadRequest, ErrUnexpected)
+		}
```

Comment:
well, if you're throwing an unexpected error, why would you say it's the client's fault?